services/src/main/java/org/fao/geonet/api/es/EsHTTPProxy.java

```diff
@@ -646,7 +646,8 @@ private void processResponse(ServiceContext context, HttpSession httpSession,
                     addRelatedTypes(doc, relatedTypes, context);
                 }
 
-                if (doc.has("_source")) {
+                if (doc.has("_source") &&
+                    doc.get("_source").has("documentStandard")) {
```

Comment:
If `documentStandard` is mandatory, would not be better to update the following method to include it always?  

https://github.com/geonetwork/core-geonetwork/blob/de2ae7457a84e37dad21a5777e88546ba0bb6358/services/src/main/java/org/fao/geonet/api/es/EsHTTPProxy.java#L426-L432

Reply:
it is already, documentStandard is SCHEMA. https://github.com/geonetwork/core-geonetwork/blob/main/core/src/main/java/org/fao/geonet/constants/Geonet.java#L635

Updated to use the index field names instead of String.